sentry-spring-boot-jakarta/src/main/java/io/sentry/spring/boot/jakarta/SentryAutoConfiguration.java

```diff
@@ -164,7 +164,7 @@ static class OpenTelemetryLinkErrorEventProcessorConfiguration {
     }
 
     @Configuration(proxyBeanMethods = false)
-    @Import(SentryGraphqlConfiguration.class)
```

Comment:
Can the `SentryGraphqlConfiguration` class be deleted, or is it still of use?

Reply:
What's the reason why we didn't update the `SentryGraphqlConfiguration` class directly?
What does it change for someone manually using the `SentryGraphqlConfiguration`?
genuinely asking as i'm not so much into backend

Reply:
I kept `sentry-spring/SentryGraphqlConfiguration` for people using Spring (not Boot) and I wasn't able to update because `SentryProperties` lives in `sentry-spring-boot` so I decided to simply duplicate the config and use the one in `sentry-spring-boot` for auto config of Spring Boot. LMK if you can think of a simpler way to achieve what we want.